docs/tutorials.md

```diff
@@ -2,28 +2,460 @@
+| `Object-detection-webcam <notebooks/401-object-detection-with-output.html>`__ |br| |n401|                                     | Object detection with a webcam or video file.                                                                                              | |n401-img1|                               |
++-------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------+
+| `Pose-estimation-webcam <notebooks/402-pose-estimation-with-output>`__ |br| |n402|                                            | Human pose estimation with a webcam or video file.                                                                                         | |n402-img1|                               |
++-------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------+
+| `Action-recognition-webcam <notebooks/403-action-recognition-webcam-with-output.html>`__ |br| |n403|                          | Human action recognition with a webcam or video file.                                                                                      | |n403-img1|                               |
++-------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------+
+
+-------------------
+
+.. note::
+   If there are any issues while running the notebooks, refer to the **Troubleshooting** and **FAQ** sections in the `Installation Guide <notebooks-installation.html>`__ or start a GitHub
+   `discussion <https://github.com/openvinotoolkit/openvino_notebooks/discussions>`__.
+
+
+
+.. raw:: html
+
+   <a name='-additional-resources' id='-additional-resources'/>
+
+`Additional Resources`_
+-------------------------
+
+* `OpenVINO™ Notebooks - Github Repository <https://github.com/openvinotoolkit/openvino_notebooks/blob/main/README.md>`_
+* `Binder documentation <https://mybinder.readthedocs.io/en/latest/>`_
+
+
+.. raw:: html
+
+   <a name='-contributors' id='-contributors' />
+
+`Contributors`_
+--------------------------
+
+|contributors|
+
+Made with `contributors-img <https://contrib.rocks>`__.
+
+
+.. |br| raw:: html
+
+   <br />
+
+.. |chinese-text| raw:: html
+
+   <span style="font-size:10px">的人不一了是他有为在责新中任自之我们</span>
```

Comment:
wth is this???

Reply:
It's an alias for a directive that injects raw html code. In this case it is a representation of the output text that has been produced by OCR ran by the notebook and is displayed in here:

![image](https://user-images.githubusercontent.com/101244613/190338776-dedd533b-188f-453c-a1d3-852bcbd3a3f7.png)

So we have a representation of an input (text in image) and an output (the text) :relaxed:

Are you worried about the meaning of the text?

Reply:
aha that explains.....the text in that image is too small for me to tell the correctness of this result, but regarding the Chinese here, I can only say: it's not a valid phrase or sentence. (I know each word but I don't understand any of them when they are put together in this way)  :)